pkg/envoy/config.go

```diff
@@ -1,9 +1,9 @@
 package envoy
 
-import k8s "github.com/openservicemesh/osm-health/pkg/kubernetes"
+import v1 "k8s.io/api/core/v1"
 
 // GetEnvoyConfig returns the Envoy config for the given pod.
-func GetEnvoyConfig(pod k8s.Pod) (*Config, error) {
+func GetEnvoyConfig(pod *v1.Pod) (*Config, error) {
```

Comment:
yay looks like this will fix my CI issues too!

Reply:
Super sorry about this breakage!